spec/services/calculation_orchestrator_spec.rb

```diff
@@ -74,6 +87,15 @@
       },
     }
   end
+  let(:capped_uplift) do
+    {
+      uplift: {
+        participants: 10_000,
+        per_participant: 100.0,
+        subtotal: 99_500.0,
+      },
+    }
+  end
```

Comment:
I don't understand where these values are coming from, and how they relate to the dummy aggregator.

Does this test belong in this file, or should it live in the spec for uplift fees calculation?